### ISSUE TITLE
Not updating

### ISSUE THREAD

**Issue by nuentes** [ronivay/xen-orchestra-docker#17]
I noticed the last 3 workflows have failed, and my container now hasn't updated in nearly a month. Can you take a look?

**Comment by ronivay** [comment]
Hi,

thanks. For some reason i haven’t received notifications for these. I’ll take a look asap.

**Comment by ronivay** [comment]
Yep it seems that arm64 build is failing and therefore the entire workflow fails and not updated images are pushed.

error:
```
#12 54.51 /usr/bin/ld: /etc/xen-orchestra/node_modules/fuse-shared-library-linux/libfuse/lib/libfuse.so: error adding symbols: file in wrong format
```

Looks like XO team is aware of this and there is a related PR open already: https://github.com/vatesfr/xen-orchestra/pull/6431

I need to disable arm64 arch from the workflow for now.

**Comment by ronivay** [comment]
There's now a new latest/5.103.1 tag available for `linux/amd64`. I'll keep this issue open once the arm64 build works again.

**Comment by nuentes** [comment]
Much appreciated!

**Comment by ronivay** [comment]
Seems that the linked PR was abandoned. I wouldn't keep hopes up that this will get fixed any time soon.

**Comment by stoneobscurity** [comment]
any update on this?

**Comment by ronivay** [comment]
Not really, keeping it here as a reference for now. I'm afraid it can't be fixed in this build if root cause is incompatible XO dependency.

**Comment by AndreiTelteu** [comment]
I have a fork with working ARM build, tested on my Raspberry PI.
https://github.com/AndreiTelteu/xen-orchestra-docker-arm
Is anyone interested in testing and feedback ?
If it works well I would make a PR to fix this here.

**Comment by boretom** [comment]
> I have a fork with working ARM build, tested on my Raspberry PI. https://github.com/AndreiTelteu/xen-orchestra-docker-arm Is anyone interested in testing and feedback ? If it works well I would make a PR to fix this here.

I just installed it on my Macbook Air M2 using @AndreiTelteu repo and the docker-compose file that came with it, it works flawless so far! Not compiled it myself yet.

Thanx @AndreiTelteu, excellent work.
